src/test/regress/expected/multi_mx_modifying_xacts.out

```diff
@@ -124,7 +124,7 @@ EXCEPTION
 END $$;
 NOTICE:  caught not_null_violation
 COMMIT;
-ERROR:  could not commit transaction for shard 1220100 on any active node
+ERROR:  was not able to make changes to shard 1220100 on any node
```

Comment:
The important part is s/commit transaction for/make changes to/. I can change the was back to could if you want... but can we come up with a final list of changes which should be made and get this merged before too many tests happen?